nmspy/data/structs.py

```diff
@@ -739,19 +957,46 @@ class cGcTechnology(ctypes.Structure):
         ("mColour", common.Colour),
         ("miLevel", ctypes.c_int32),
         ("mbChargeable", ctypes.c_ubyte),
+        ("_padding0x235", ctypes.c_ubyte * 0x3),
```

Comment:
this isn't need since the proceeding field will get aligned at 4 bytes.
You only need padding before fields which will have an align of 1 when it should be something higher (as is the case of `TkID`'s)